edit/kubevirt.io.virtualmachine/index.vue

```diff
@@ -457,10 +467,14 @@ export default {
           <NodeScheduling :mode="mode" :value="spec.template.spec" :nodes="nodesIdOptions" />
         </Tab>
 
+        <Tab v-if="isEdit" :label="t('harvester.tab.accessCredentials')" name="accessCredentials" :weight="-4">
+          <AccessCredentials v-model="accessCredentials" :mode="mode" :resource="value" :is-qemu-installed="isQemuInstalled" />
```

Comment:
You can't get `isQemuInstalled` here, you should add a computed property.
```
computed: {
  isQemuInstalled() {
    return this.value.isQemuInstalled
  }
}
```

Reply:
Done.